nginx/nginx.conf

```diff
@@ -0,0 +1,19 @@
+server {
+    listen 80 default_server;
+    root /home/restapi/http/public;
+
+    index index.php index.html index.htm;

```

Comment:
I'm not sure if adding more than one index name has any sense. Just put here your real index name.